src/styles/page-template/index.md.njk

```diff
@@ -132,12 +132,14 @@ To change the components that are included in the page template by default, set
         Add content just before the closing <code>&lt;/body&gt;</code> element.
       </td>
     </tr>
-    
+
     <tr class="govuk-table__row">
       <td class="govuk-table__cell">bodyStart</td>
       <td class="govuk-table__cell">Block</td>
       <td class="govuk-table__cell">
         Add content after the opening <code>&lt;body&gt;</code> element.
+        <br>
+        For example: The cookie banner component.
```

Comment:
We need the URL to link from the words 'cookie banner' to the guidance.

Reply:
@EoinShaughnessy Linking to `/components/cookie-banner/` should work, as that's what the URL will be once it's published 🙂